lib/Interpreter/CIFactory.cpp

```diff
@@ -42,9 +43,19 @@
 
 #include <memory>
 
+#ifndef _MSC_VER
+#include "cling-compiledata.h"
+# include <unistd.h>
+# define getcwd_func getcwd
+#endif
+
+// FIXME: This code has been taken (copied from) llvm/tools/clang/lib/Driver/WindowsToolChain.cpp
```

Comment:
Indeed, in order to do that we could rely on `getToolChain()`. This way we could keep the implementation in one place. Last time @bellenot tried it seemed quite a bit of coding, plus eventually patching clang and we decided to postpone it.